src/utils.ts

```diff
@@ -93,3 +93,8 @@ export function isWithinRootPath(file: string) {
 
 	return workspace.rootPath != null && file.startsWith(workspace.rootPath + path.sep); 
 }
+
+export function log(message: any): void {
+	if (config.verbose)
```

Comment:
I have no problem dropping `verbose` :) I'll keep this PR as is and we can remove in future work.